src/wp-admin/includes/media.php

```diff
@@ -2285,7 +2286,7 @@ function media_upload_form( $errors = null ) {
 			_ex( 'Upload', 'verb' );
 			?>
 		</label>
-		<input type="file" name="async-upload" id="async-upload" required />
+		<input type="file" name="async-upload" id="async-upload"<?php echo $required; ?>/>
```

Comment:
This would remove the space before the slash `/>`:
- `id="async-upload" required/>` on the New Media screen
- `id="async-upload"/>` in other contexts

If the space is moved after 'required' in the variable, I think having the space before the PHP tag is easier to read than placing the space before the slash.
```suggestion
		<input type="file" name="async-upload" id="async-upload" <?php echo $required; ?>/>
```

Reply:
Thanks for the feedback. I made the changes locally before noticing your suggested changes! I think the result is the same, checking